src/main/java/bisq/desktop/main/dao/proposal/ProposalDetailsWindow.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.desktop.main.dao.proposal;
+
+import bisq.desktop.main.overlays.Overlay;
+import bisq.desktop.util.BsqFormatter;
+import bisq.desktop.util.Layout;
+
+import bisq.core.btc.wallet.BsqWalletService;
+import bisq.core.dao.proposal.ProposalPayload;
+import bisq.core.locale.Res;
+
+import javafx.geometry.Insets;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static bisq.desktop.util.FormBuilder.addButtonAfterGroup;
+
+public class ProposalDetailsWindow extends Overlay<ProposalDetailsWindow> {
+    protected static final Logger log = LoggerFactory.getLogger(ProposalDetailsWindow.class);
+    private final BsqFormatter bsqFormatter;
+    private final BsqWalletService bsqWalletService;
+    private ProposalPayload proposalPayload;
+    private ProposalDisplay proposalDisplay;
+
+
+    ///////////////////////////////////////////////////////////////////////////////////////////
+    // Public API
+    ///////////////////////////////////////////////////////////////////////////////////////////
+
+    public ProposalDetailsWindow(BsqFormatter bsqFormatter, BsqWalletService bsqWalletService, ProposalPayload proposalPayload) {
```

Comment:
Nit: Please wrap lines at 120 chars. I'm mentioning this just once in this review, but there are multiple places where this nit applies. Note that I see in many places you _did_ wrap at 120, thanks.

Reply:
It seems it only applies the wrapping if you touch the code at that line. Reformatting does not apply it over the whole class.

Reply:
@sqrrm I merge as I am not sure that the already merged core will be compatible with the old desktop master.